projects/plugins/crm/tests/php/rest-api/v4/class-dummy-test.php

```diff
@@ -0,0 +1,29 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+/**
+ * Base class for API test cases.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack_CRM\Tests;
+
+use WorDBless\BaseTestCase;
+
+/**
+ * Dummy test class to ensure tests are working.
+ *
+ * @covers Automattic\Jetpack_CRM
+ */
+class Dummy_Test extends BaseTestCase {
+
+	/**
+	 * Dummy test to ensure PHPUnit works.
+	 */
+	public function test_that_our_tests_are_testing_the_test_system() {
+		add_option( 'jpcrm_test_option', 'hello' );
+
+		$this->assertSame( 'hello', get_option( 'jpcrm_test_option' ) );
+	}
```

Comment:
This test class is simply to verify that everything works, so you don't need to focus on the test itself 😄

Reply:
👍